requirements-sdk.in

```diff
@@ -7,4 +7,5 @@
 # Also avoid putting any libraries that are only used by the build system (directly or
 # indirectly).  Those belong in "requirements-build.in".
 websockets
-aiohttp
\ No newline at end of file
+aiohttp
+urllib3>=2.2.2
```

Comment:
In general, I like to avoid pinned requirements like this in the `.in` files unless a standard upgrade would get something bad.  Here I think we could just update the `requirements.txt` file directly since if/when we did do a full upgrade of the requirements we'd get the newer version anyway.  Which also begs the question of where the change to the `requirements.txt` file is.  That file is also under SCC, so it should be in the PR, right?

Reply:
regeneration seemed to have missed it.  Redid it all anyway -- should be better now.